<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackHeader_UI</class>
 <widget class="QWidget" name="TrackHeader_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>86</width>
    <height>39</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="track_number">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="effect_label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="buttonLock">
     <property name="maximumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string notr="true">L</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="buttonAudio">
     <property name="maximumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string notr="true">A</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="buttonVideo">
     <property name="maximumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string notr="true">V</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
